third-party/llvm/Makefile

```diff
@@ -41,9 +38,6 @@ clobber: FORCE
 
 # -enable-debug-runtime --enable-debug-symbols creates huge binaries
 #  so we by default do not include LLVM debugging information.
-# Note that the configured-clang-sysroot-arguments could possibly be added
-# to LLVM as configure options (--with-clang-resource-dir,
-# --with-c-include-dirs, --with-default-sysroot)
```

Comment:
I think this comment is worth keeping but maybe just change it to "as cmake options."

Reply:
Sounds good.